server/src/main/java/com/google/coffeehouse/common/StorageHandler.java

```diff
@@ -14,40 +14,109 @@
 
 package com.google.coffeehouse.common;
 
+import static com.google.cloud.spanner.TransactionRunner.TransactionCallable;
+
 import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.Mutation;
 import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.Spanner;
 import com.google.cloud.spanner.Statement;
+import com.google.cloud.spanner.TransactionContext;
+import com.google.cloud.spanner.Value;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 /**
 * The StorageHandler class holds the query functions to get information from the database.
 */
 public class StorageHandler {
   public static final String NO_PRONOUNS = "No pronouns";
   public static final String PERSON_DOES_NOT_EXIST = "This person does not exist in the database.";
-  public static final String ERROR_MORE_THAN_ONE_PERSON = "ERROR: More than one person per user ID.";
+  public static final String ERROR_MORE_THAN_ONE_PERSON = "More than one person per user ID.";
 
   public static final String CLUB_DOES_NOT_EXIST = "This club does not exist in the database.";
-  public static final String ERROR_MORE_THAN_ONE_CLUB = "ERROR: More than one club per club ID.";
+  public static final String ERROR_MORE_THAN_ONE_CLUB = "More than one club per club ID.";
 
   public static final String NO_AUTHOR = "No author";
   public static final String NO_ISBN = "No ISBN";
   public static final String BOOK_DOES_NOT_EXIST = "This book does not exist in the database.";
-  public static final String ERROR_MORE_THAN_ONE_BOOK = "ERROR: More than one book per book ID.";
+  public static final String ERROR_MORE_THAN_ONE_BOOK = "More than one book per book ID.";
+  
+  public static Person createPersonFromId(String userId) {
+    Spanner spanner = StorageHandlerSetup.createSpannerService();
+    DatabaseClient dbClient = StorageHandlerSetup.createDbClient(spanner);
+    Person person = StorageHandler.getPersonQuery(dbClient, userId);
+    return person;
+  }
+
+  public static Club createClubFromId(String clubId) {
```

Comment:
This seems to be retrieving an existing club instead of creating a new one. If that is the case, consider renaming the method to something like `getClubById` or `fetchClubById`. I would expect a call with this name to add the club to the database.

Reply:
This creates a club object by retrieving an existing club from the database, so I'll rename the function to be `fetchClubById` instead.